website/config.yaml

```diff
@@ -66,8 +66,9 @@ params:
         url: 'https://slack.k8s.io/'
         icon: fab fa-slack
         desc: 'Chat with us on Slack in the #aws-provider channel'
-  latest_release_version: v0.9.1
+  latest_release_version: v0.9.2
   versions:
+    - "v0.9.2"
     - "v0.9.1"
     - "v0.8.2"
     - "v0.7.3"
```

Comment:
Can you please remove a few of the older versions here? the website is becoming too large.